src/pages/Organization/components/GovtOrganizationSelector.tsx

```diff
@@ -89,24 +95,48 @@ function OrganizationLevelSelect({
 export default function GovtOrganizationSelector(
   props: GovtOrganizationSelectorProps,
 ) {
-  const { onChange, required, selected, authToken } = props;
+  const {
+    onChange,
+    required,
+    selected,
+    authToken,
+    navOrganizationId,
+    organizationId,
+  } = props;
   const [selectedLevels, setSelectedLevels] = useState<Organization[]>([]);
 
+  const { organizations: navOrg, isFetching: fetchingNavOrg } =
+    useGovtOrganizationLevel({
+      index: 0,
+      onChange: () => {},
+      parentId: "",
+      authToken,
+    });
+
+  const { organizations: org, isFetching: fetchingOrg } =
+    useGovtOrganizationLevel({
+      index: 1,
+      onChange: () => {},
+      parentId: navOrganizationId || "",
+      authToken,
+    });
+
   useEffect(() => {
-    if (selected && selected.length > 0) {
-      let currentOrg = selected[0];
-      if (currentOrg.level_cache === 0) {
-        setSelectedLevels(selected);
-      } else {
-        const levels: Organization[] = [];
-        while (currentOrg && currentOrg.level_cache >= 0) {
-          levels.unshift(currentOrg);
-          currentOrg = currentOrg.parent as unknown as Organization;
-        }
-        setSelectedLevels(levels);
-      }
+    if (selected?.length) {
+      setSelectedLevels(selected);
+      return;
+    }
+    const selectedOrgs: Organization[] = [];
```

Comment:
So when you change an existing component, make sure to test to existing use cases as well.